devassistant/actions.py

```diff
@@ -561,67 +553,64 @@ class AutoCompleteAction(Action):
     args = [argument.Argument('path', 'path', default='', nargs='?')]
     hidden = True
 
-    # Assistant names are hardcoded here because on the root level, we only
-    # want to show these nice long forms. All forms incl. old aliases are
-    # autocompleted, of course.
-    _assistant_names = ['create', 'tweak', 'prepare', 'extra']
-    _assistants = bin.TopAssistant().get_subassistants()
-    _actions = [action for action in actions if not action.hidden]
-    _special_tokens = ['--debug']
-
-    @classmethod
-    def run(cls, **kwargs):
-        path = kwargs.get('path', '').split()
-
-        flags = cls._get_flags_for_path(path)
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+        self._assistant_names = ['create', 'tweak', 'prepare', 'extra']
```

Comment:
Things in here, do we need to distinguish them per instance?

(also for some of the next lines)